src/web/app/components/teammates-common/question-type-id.pipe.ts

```diff
@@ -0,0 +1,43 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { FeedbackQuestionType } from '../../../types/api-output';
+import {
+  QuestionsSectionQuestions,
+ } from '../../pages-help/instructor-help-page/instructor-help-questions-section/questions-section-questions';
+
+/**
+ * Pipe to get Question ID from {@code QuestionsSectionQuestions}.
```

Comment:
Perhaps we can rename this file to question-type-help-path to be more descriptive?

Reply:
Thats true, will update